README.md

```diff
@@ -1,2 +1,171 @@
-# LeaderBoard-Project
-LeaderBoard Project
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖-About-the-Project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖-About-the-Project <a name="about-project"></a>
+
+![screenshot 1](image1.jpg)
+
+-***Live Demo Link***-coming soon
+
+**LeaderBoard Project** -In this activity I have set up a JavaScript project for the Leader board list app, using webpack and ES6 features, notably modules.
+In Phase 1 branch of this repository, I have used Gitflow.I have created an npm project with webpack.I wrote plain HTML markup with minimum styling (just to make the layout) and I have used ES6 modules, with import and export.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+
+### Tech Stack <a name="tech-stack"></a>
+- Git and Github
+- VS-code
+- JavaScript
+- HTM
+- CSS
+- Webpack
+
+### Key Features <a name="key-features"></a>
+
+- **Use Webpack**
+- **Use ES6**
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">JavaScript</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [LIVE-DEMO](<a href="#">(https://github.com/harshi0102/LeaderBoard-Project.git)
```

Comment:
- Hi, your project is not deployed yet, and you are using a fake link. I would advise you to remove this section if you are not going to deploy or leave a notification like `coming soon`. This is because it will make your Readme template look more professional.